templates/Coaster/show.html.twig

```diff
@@ -10,7 +10,7 @@
     {# <meta name="description" content="{{ coaster.name }} is a {{ coaster.manufacturer.name }} roller coaster"/> #}
 {# {% endblock %} #}
 
-{% set isImperial = units.isImperial(app.request.locale) %}
+{% set isImperial = units.isImperial(app.request.locale, app.user) %}
```

Comment:
Can you remove this and inject dependencies at the service level
Example: https://stackoverflow.com/questions/39871558/how-to-get-the-current-user-in-twig-extension-service-symfony2-8/39876994#39876994

Reply:
Made the changes. Can you check?